mlir/utils/jenkins/Dockerfile

```diff
@@ -68,22 +68,16 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   libnuma-dev \
   software-properties-common
 
-RUN apt-add-repository ppa:deadsnakes/ppa && apt-get update \
-     && apt-get install -y --no-install-recommends \
-            python3.8 python3.8-dev python3.8-venv python3-psutil \
-            python3-pip python3-setuptools \
-     && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 10 \
-     && apt upgrade \
-     && python3 -m pip install --upgrade pip \
-     && pip3 install pandas numpy scipy jinja2 tomli \
-     && pip3 install -r requirements.txt \
-     && sed -i -e 's#/usr/bin/python3#/usr/bin/python3.6#' /usr/bin/apt-add-repository /usr/bin/add-apt-repository
+RUN apt-get update && apt-get install -y --no-install-recommends \
+           python3.8 python3.8-dev python3.8-venv python3-psutil \
```

Comment:
What do you mean by "this chunk"?

Reply:
The whole `RUN` command.

Reply:
Do you know who is the consumer of all those dependencies? All in our scripts?

It is strange that we'd call out a particular 3.8 to be the python3 version we use. Do you think we should use python3 instead overall?

Reply:
We needed to bump to 3.8 because our test scripts used `async`, which landed in 3.7 but Ubuntu 18.04 only had Python 3.6.

I'm pretty sure `-psutil` and maybe  `-venv` are from upstream.

We can probably drop the explicit install of Python.

The install of `pandas`, `numpy`, and so on are needed by the performance scripts

Reply:
`python3-venv` is required by MIOpen. Others are used by performance report generator. `tomli` is used to auto generate rock E2E test suits. Not sure about `jinja2`, which does appear in our codebase.

Reply:
jinja2 is, IIRC, an indirect dependency of Pandas's HTML support but I'd need to poke Google there

Reply:
@zhanglx13 In which stage does MIOpen need `python3-venv`, but not `python3.8-venv`?

Reply:
`python3-venv` will be `python3.8-venv` on 20.04